static/js/settings_account.js

```diff
@@ -446,9 +469,11 @@ export function set_up() {
         const opts = {
             success_continuation() {
                 channel.set_password_change_in_progress(false);
-                overlays.close_modal("#change_password_modal");
+                dialog_widget.close_modal();
             },
             error_continuation() {
+                dialog_widget.hide_dialog_spinner();
+                change_password_error.show();
```

Comment:
Can you verify whether we require this statement to show the error element here or is it handled by `ui_report` function itself?

Reply:
Fixed